test/fixtures/banking.ofx.sgml

```diff
@@ -45,7 +45,7 @@ NEWFILEUID:NONE
           <STMTTRN>
             <TRNTYPE>PAYMENT</TRNTYPE>
             <DTPOSTED>20070606120000.000</DTPOSTED>
-            <TRNAMT>-11.11</TRNAMT>
+            <TRNAMT>-.11</TRNAMT>
```

Comment:
It's probably cleaner to add another line to the existing `test_for_pennies` test rather than change fixture data.